packages/@aws-cdk/aws-neptune-alpha/test/integ.cluster.ts

```diff
@@ -37,6 +38,7 @@ const cluster = new DatabaseCluster(stack, 'Database', {
   vpc,
   vpcSubnets: { subnetType: ec2.SubnetType.PRIVATE_WITH_EGRESS },
   instanceType: InstanceType.R5_LARGE,
+  engineVersion: EngineVersion.V1_2_1_0,
```

Comment:
If `engineVersion` is not specified, CFN will create the cluster using the latest engine version. Currently, the latest engine is the v1.3 series, which the existing CDK code does not support. 
(If the parameter group is also not set to v1.3, the deployment will fail.)

Therefore, in this integration test, the engine and parameter group have been explicitly set to v1.2.

Reply:
I plan to address the support for the v1.3 in a separate issue.

Reply:
Sounds good, feel free to ping me for a review when you get around to that.

Reply:
This problem is resolved by #28647.
Do I need to conduct this integration test with version 1.3, or is it okay to proceed as it is?